src/types/projects.ts

```diff
@@ -16,3 +16,25 @@ export type ProjectProfessionalResponse = {
   professional: ProfessionalResponse;
   responsibility: string;
 };
+
+export type ParsedProjects = {
+  id: string;
+  name: string;
+  from: string;
+  to: string;
+  duration: string;
+  description: string;
+  professionals: ProjectProfessionalResponse[];
+  attributes: ProjectAttributeResponse[];
+};
+
+export type ProjectRequest = {
+  id: string;
+  name: string;
+  from: string;
+  to: string;
+  duration: string;
+  description: string;
+  professionals: ProjectProfessionalResponse[];
+  attributes: ProjectAttributeResponse[];
```

Comment:
```suggestion
  attributes: { from: Date; to: Date; attributeId: string }[];
```
Here is the reason of why we dont need "type" on "ParsedProjectAttribute". Because the backend is expecting this structure,